python/py-nbxmpp/Portfile

```diff
@@ -1,35 +1,39 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
-PortGroup           python 1.0
+PortSystem           1.0
+PortGroup            python 1.0
+PortGroup            gitlab 1.0
 
-name                py-nbxmpp
-version             2.0.6
-revision            0
-categories-append   net
-platforms           darwin
-license             GPL-3
-supported_archs     noarch
+gitlab.instance      https://dev.gajim.org
+gitlab.setup         gajim python-nbxmpp 3.0.1 nbxmpp-
 
-python.versions     27 37 38 39 310
+name                 py-nbxmpp
+version              3.0.1
+revision             0
+categories-append    net chat
+license              GPL-3
+supported_archs      noarch
+maintainers          nomaintainer
 
-maintainers         nomaintainer
+description          Non blocking Jabber/XMPP module
 
-description         Non blocking Jabber/XMPP module
+long_description     The goal of this python library is to provide a way \
+                     for python applications to use Jabber/XMPP networks in a non-blocking \
+                     way. Originally, a fork of the xmpppy jabber python library.
 
-long_description    The goal of this python library is to provide a way for \
-                    python applications to use Jabber/XMPP networks in a non-blocking \
-                    way. Originally, a fork of the xmpppy jabber python library.
+checksums    rmd160  069352f7ead2dd75a9c1ca6a9373572afeea4833 \
+             sha256  ad5fca43c902d7ae2317a628062eda08ec0c1ebbbd3066e43cfbd536fa6fef64 \
+               size  126138
```

Comment:
why is the alignment here messed up?

Reply:
> why is the alignment here messed up?

Changed to your preferred style.

Reply:
just to be clear: it's not my personally preferred style, it's what the MacPorts' project wants and as the modeline dictates 😉